services/testing-framework/.dockerignore

```diff
@@ -17,4 +17,5 @@ services/kuber
 .gitattributes
 manifests
 docs
-docker-compose.yml
\ No newline at end of file
+docker-compose.yml
+# REMOVE BEFORE MERGING. Added comment to trigger CI
```

Comment:
You probably wanted to remove this change

Reply:
yes, this will be remove after approval